### PR TITLE
(maint) Make some small updates to the README

### PR DESCRIPTION
**Pull request by HAIL9000**
Update the README so that it is accurate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ It can also print the JSON to stdout:
 $ puppet strings some_manifest.pp --emit-json-stdout
 ```
 
-The schema for the JSON which Strings emits is [well documented](json_dom.md).
+The schema for the JSON which Strings emits is [well documented](https://github.com/puppetlabs/puppetlabs-strings/blob/master/json_dom.md).
```
Comment (iankronquist) [inline README.md]: Oops, sorry, should have tested this. I just assumed a relative link would work.

**Comment by iankronquist** [comment]
Looks good to me.

**Comment by HAIL9000** [comment]
It does work on GitHub, but I wasn't sure about the forge. Better safe than sorry!

File: README.md (hunk 1 above)
Comment (HAIL9000) [inline README.md]: It does work on GitHub, but I wasn't sure about the forge. Better safe than sorry!

**Pull request merged**